tor/core/inbox.py

```diff
@@ -77,27 +77,33 @@ def process_reply(reply, config):
             reply.mark_read()
             return
 
-        if 'i accept' in reply.body.lower():
+        r_body = reply.body.lower()  # cache that thing
+
+        if 'i accept' in r_body:
             process_coc(reply, config)
             reply.mark_read()
             return
 
-        if 'claim' in reply.body.lower():
+        if 'claim' in r_body:
             process_claim(reply, config)
             reply.mark_read()
             return
 
-        if 'done' in reply.body.lower():
-            process_done(reply, config)
+        if (
+            'done' in r_body or
+            'deno' in r_body  # we <3 u/Lornescri
+        ):
+            alt_text = True if 'done' not in r_body else False
```

Comment:
Should this be changed? We're checking for done in `r_body` twice. Maybe break up the if statement or something.

Not really a big deal I guess.

Reply:
They do different things. One is part of the generic "do we follow this path" logic, the second is "no matter how many alternate routes we have, if that route we took to get here is not `done` then set a boolean".

Reply:
Oh yeah good point.